draft-irtf-cfrg-hpke.md

```diff
@@ -885,6 +894,45 @@ shared secret is the all-zero value and abort if so, as described in
 | 0x0002 | HKDF-SHA384 | 48  | {{?RFC5869}} |
 | 0x0003 | HKDF-SHA512 | 64  | {{?RFC5869}} |
 
+### Input Length Restrictions {#kdf-input-length}
+
+This document defines `LabeledExtract` and `LabeledExpand` based on the
+KDFs listed above. These functions add prefixes to their respective
+inputs `IKM` and `info` before calling the KDF's Extract and Expand
+functions. This leads to a reduction of the maximum input length that
+is available for the inputs `psk`, `pskID`, `info`, `exporter_context`.
+The following table lists the maximum allowed lengths of these fields
+for the KDFs defined in this document, as inclusive bounds in bytes:
+
+| Input            | HKDF-SHA256  | HKDF-SHA384   | HKDF-SHA512   |
+|:-----------------|:-------------|:--------------|:--------------|
+| psk              | 2^{61} - 81  | 2^{125} - 145 | 2^{125} - 145 |
+| pskID            | 2^{61} - 83  | 2^{125} - 147 | 2^{125} - 147 |
+| info             | 2^{61} - 82  | 2^{125} - 146 | 2^{125} - 146 |
+| exporter_context | 2^{61} - 111 | 2^{125} - 191 | 2^{125} - 207 |
```

Comment:
While I guess it's worth knowing that these limits exist, they seem ... unlikely to ever be hit in reality.

Reply:
That's right. I added a couple of words to put it into perspective.